app/views/channelcomparison.php

```diff
@@ -30,21 +30,26 @@
 
 <?php
 
-echo "\n<table class='collapsable'>" .
-     "  <tr>\n" .
-     "    <th colspan='3'>Locale: {$locale}</th>\n" .
-     "  </tr>\n" .
-     "  <tr>\n" .
-     "    <th>Key</th>\n" .
-     "    <th>{$chan1}</th>\n" .
-     "    <th>{$chan2}</th>\n" .
-     "  </tr>\n";
+if (empty($common_strings)) {
+    echo "<h3>Comparison is empty</h3>\n" .
+         "<p class='subtitle'>There are no strings to compare: this locale is not supported for both channels.</p>\n";
```

Comment:
If the strings are identical for the locale on two repositories (beta and release for example), then the array will be empty, yet the locale is supported on both channels. Ex Acholi:
https://transvision.mozfr.org/channelcomparison/?locale=ach&chan1=beta&chan2=release